src/CUSOLVERRF/CUSOLVERRF.jl

```diff
@@ -76,6 +76,19 @@ function glu_analysis_host(
             n, nnzA, desca,
             h_rowsA, h_colsA, h_Qreorder,
         )
+    elseif ordering == :MDQ
+        CUSOLVER.cusolverSpXcsrsymmdqHost(
+            spH,
+            n, nnzA, desca,
+            h_rowsA, h_colsA, h_Qreorder,
+        )
+
+    elseif ordering == :METIS
+        CUSOLVER.cusolverSpXcsrmetisndHost(
+            spH,
+            n, nnzA, desca,
+            h_rowsA, h_colsA, C_NULL, h_Qreorder,
```

Comment:
Why do we need to pass `C_NULL` there?

Reply:
CUSOLVER allows to pass some options to METIS (https://docs.nvidia.com/cuda/cusolver/index.html#cusolver-lt-t-gt-csrmetis) but by default `options = C_NULL`.